packages/shared/src/lib/tracking.ts

```diff
@@ -182,6 +202,7 @@ const createEventFromTracking = (action: OphanAction) => {
                 labels,
             },
             ...(abTest ? { abTest } : {}),
+            ...(amountsAbTest ? { amountsAbTest } : {}),
```

Comment:
The `amountsAbTest` field here will not be picked up by ophan as it's not part of the expected model.
We need to instead send separate ophan events for the choice cards sub-component

Reply:
agreed. Will implement